ExportShared.as

```diff
@@ -0,0 +1,21 @@
+shared enum ActiveCam {
+	None = 0,
+	Cam1 = 1,
+	Cam2 = 2,
+	Cam3 = 3,
+	Cam1Alt = 4,
+	Cam2Alt = 5,
+	Cam3Alt = 6,
+	FreeCam = 7,
+	Backwards = 8,
+	EditorOrbital,
+	Orbital3d,
+	Helico,
+	HmdExternal,
+	ThirdPerson,
+	FirstPerson,
+	Target,
+	Cam0,
+	Other,
+	Loading
+}
```

Comment:
Is there no enum in the game for this? (Anywhere in `Openplanet.h`?) This will probably also be different on Maniaplanet and Turbo.

Additionally, this should be in the `Camera` namespace.

Reply:
No enum for this exactly -- I did however find the following. Curiously, these values line up with some other's that I've seen (particularly I remember 18 for cam 1 somewhere), so that might be a better source. It might have been in game terminal. 

```c++
  enum class CGameItemModel::EnumDefaultCam {
    None = 0,
    Default = 1,
    Free = 2,
    Spectator = 3,
    Behind = 4,
    Close = 5,
    Internal = 6,
    Helico = 7,
    FirstPerson = 8,
    ThirdPerson = 9,
    ThirdPersonTop = 10,
    Iso = 11,
    IsoFocus = 12,
    Dia3 = 13,
    Board = 14,
    MonoScreen = 15,
    Rear = 16,
    Debug = 17,
    _1 = 18, // 1
    _2 = 19, // 2
    _3 = 20, // 3
    Alt1 = 21,
    Orbital = 22,
    Decals = 23,
    Snap = 24,
    NearOpponents = 25,
    MapThumbnail = 26,
  };
  ```
  
there's also `CGameCtnMediaBlockCameraGame::EGameCam` but not really relevant and only 7 options

Reply:
If there's really no other enum available for this, it makes sense to keep. 👍